site/_apps/preact-hacker-news.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 16
```

Comment:
Minor nitpick: could you update the weights to show after [the most recent app](https://github.com/tastejs/hacker-news-pwas/commit/c54aaff35cb5adae88b8ad749b8c1d505ddd3289) (this should be the 22nd)